contexts/useWallet.d.tsx

```diff
@@ -16,10 +17,11 @@ export type	TBalances = {
 
 export type	TWalletContext = {
 	balances: TBalances,
+	isLoading: boolean,
 	allowances: {[key: string]: BigNumber},
 	yveCRVClaimable: TClaimable;
 	useWalletNonce: number,
-	refresh: () => Promise<void>,
+	refresh: () => Promise<{[key: string]: TBalanceData}>,
```

Comment:
This type seems to be used quite a bit, I'd consider making a type on `types/types.tsx` and export it to use across the app:


```typescript
export type TBalances = {[key: string]: TBalanceData};
```

then

```suggestion
	refresh: () => Promise<TBalances>,
```